### ISSUE TITLE
Freighter won't process a transaction on testnet 

### ISSUE THREAD

**Issue by shanzzam** [stellar/freighter#262]
### What version are you using?
1.1.3, verified via extension settings.

### What did you do?
I'm using Freighter to log in on the Account Viewer using testnet https://accountviewer.stellar.org/dashboard?testnet=true
I can log in successfully with Freighter, verify my account information and verify I'm on testnet. Freighter setting is on testnet as well.
I created a payment to another test account I've been using. It's created and active, funded with XLM.
When verifying the transaction via Freighter I receive the message:
"Freighter is currently on Test network
The transaction you’re trying to sign is on Public network.
Signing this transaction is not possible at the moment."

### What did you expect to see?
I expected the transaction to be valid and accepted as I'm on the Test network and have no other browsers open.

### What did you see instead?
I saw the message that I couldn't sign the transaction.

**Comment by shanzzam** [comment]
When trying to troubleshoot this further I used the Account Viewer on mainnet and logged in using Freighter to see what would happen. (I set Freighter to use the Public network in the Settings) Turns out I have the same account on main net. I'm not sure what the probability is of this happening and it may be a far edge case, but I would still expect my transaction on testnet o go through.

Note that my transaction was successfully signed for the Public network.
